.github/workflows/tests.yml

```
@@ -691,26 +744,34 @@ jobs:
           harpy simulate cnv --quiet 2 --prefix Simulate/cnvvcf --vcf Simulate/cnv/haplotype_1/sim.hap1.cnv.vcf  test/genome/genome.fasta.gz
+          ls -lh Simulate/cnvvcf
       - name: simulate translocations
         shell: micromamba-shell {0}
         if: always()
         run: |
           harpy simulate translocation --quiet 2 --count 10 -z 0.5 test/genome/genome.fasta.gz
+          ls -lh Simulate/translocation
           harpy simulate translocation --quiet 2 --prefix Simulate/transvcf --vcf Simulate/translocation/haplotype_1/sim.hap1.translocation.vcf  test/genome/genome.fasta.gz
-
+          ls -lh Simulate/transvcf
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)